Libraries/Inspector/ElementProperties.js

```diff
@@ -24,29 +23,21 @@ const flattenStyle = require('flattenStyle');
 const mapWithSeparator = require('mapWithSeparator');
 const openFileInEditor = require('openFileInEditor');
 
-import type {DangerouslyImpreciseStyleProp} from 'StyleSheet';
+import type {ViewStyleProp} from 'StyleSheet';
 
-class ElementProperties extends React.Component<{
+type Props = $ReadOnly<{|
   hierarchy: Array<$FlowFixMe>,
-  style?: DangerouslyImpreciseStyleProp,
-  source?: {
-    fileName?: string,
-    lineNumber?: number,
+  style?: ?ViewStyleProp,
+  source?: ?{
+    fileName?: ?string,
+    lineNumber?: ?number,
   },
-}> {
-  static propTypes = {
-    hierarchy: PropTypes.array.isRequired,
-    style: PropTypes.oneOfType([
-      PropTypes.object,
-      PropTypes.array,
-      PropTypes.number,
-    ]),
-    source: PropTypes.shape({
-      fileName: PropTypes.string,
-      lineNumber: PropTypes.number,
-    }),
-  };
+  frame?: ?Object,
+  selection?: ?number,
+  setSelection?: (?number) => mixed,
```

Comment:
setSelection is only ever called with a number, never with null, so the argument shouldn't be nullable. `setSelection?: (number) => mixed`